### ISSUE TITLE
ezyang/htmlpurifier serializer directory not writable

### ISSUE THREAD

**Issue by shochdoerfer** [passbolt/passbolt_api#274]
I just saw in our passbolt.log file the following error:
```
Warning (512): Directory /var/www/html/vendor/ezyang/htmlpurifier/library/HTMLPurifier/DefinitionCache/Serializer not writable, please chmod to 777 in [/var/www
/html/vendor/ezyang/htmlpurifier/library/HTMLPurifier/DefinitionCache/Serializer.php, line 297]
```

I'd propose to configure a folder like $PASSBOLT_DIR/tmp/htmlpurifier to store the serialized data. Storing those files deep somewhere in the vendor folder feels wrong to me.

**Comment by ThunderBirdsX3** [comment]
Any solution?

**Comment by cedricalfonsi** [comment]
@shochdoerfer @ThunderBirdsX3 
Thank you for the report.
A fix will be released with v2.4.0 (ref #PASSBOLT-3082)

**Comment by cedricalfonsi** [comment]
Solved with v2.4
